edge-modules/api-proxy-module/rust-sdk/azure-iot-mqtt/src/io.rs

```diff
@@ -4,392 +4,518 @@ use std::future::Future;
 
 /// A [`mqtt3::IoSource`] implementation used by the clients.
 pub struct IoSource {
-	iothub_hostname: std::sync::Arc<str>,
-	iothub_host: std::net::SocketAddr,
-	authentication: crate::Authentication,
-	timeout: std::time::Duration,
-	extra: IoSourceExtra,
+    iothub_hostname: std::sync::Arc<str>,
```

Comment:
is it just formatting?

Reply:
actually both: changes and formatting

Reply:
(You can append `?w=1` to the URL to ignore whitespace. ie https://github.com/Azure/iotedge/pull/4879/files?w=1 )